20230801-bucket-level-versioning-configuration.md

```diff
@@ -0,0 +1,154 @@
+tags: [ "satellite", "object versioning", "bucket level versioning" ]
+---
+
+# Object versioning configuration at the bucket level
+
+## Abstract
+
+This is a design document and implementation strategy for maintaining bucket level configuration metadata used to 
+manage versioning for objects within the bucket.
+
+## Header
+
+Date: 2023-08-16<br>

```

Comment:
I did it wrong, but I think that we should match this date with the filename date.

https://github.com/storj/design-docs/pull/9

Reply:
I don't understand, there is a new commit and the comments are resolved, but I cannot see any change. Did you miss to push them?

Reply:
Sorry, looks like I didn't commit the changes. Should be fixed now.